docs/application/dotnet/guides/nui/nui-components/CheckBox.md

```diff
@@ -1,77 +1,205 @@
 # CheckBox
-CheckBox is a common component and describes what action will occur when you select it.  
-A CheckBox can only contain an icon, and can be created using style.
 
-![CheckBox1](./media/CheckBox1.png)
+Checkbox is a UI component connected with the click events. The base class for a `CheckBox` class is a [Tizen.NUI.Components.Button](https://samsung.github.io/TizenFX/latest/api/Tizen.NUI.Components.Button.html), which properties can be used to specify the checkbox. In opposite to a button, that can contain both an icon and a text, a checkbox can only contain an icon. The default checkbox created with NUI is as follows:
```

Comment:
Checkbox is a UI component connected with the click events. The base class for a `CheckBox` class is a Tizen.NUI.Components.Button, you can use the base class properties to specify the checkbox. Unlike a button that can contain both an icon and a text, a checkbox can only contain an icon. The default checkbox created with NUI is as follows:


@mijin85cho I have removed the URL to be consistent with other pages, thoughts?
@mgrudzinska , which properties can be used to specify the checkbox. ? whose properties base class? Please check my rephrase on top?

Reply:
@annie-samsung 

The base class for a CheckBox class is a Tizen.NUI.Components.Button, you can use the base class properties to specify the checkbox.

I think the way two sentences are connected is weird. Is it OK to just add two sentences without conjunction?

The link to the related API helps user to access to more information, so you can keep it. If you are a user, the link would be helpful to you..

Reply:
> @mgrudzinska , which properties can be used to specify the checkbox. ? whose properties base class? Please check my rephrase on top?

I think the "which" means "Tizen.NUI.Components.Button", in the previous sentence.

As you can see in https://samsung.github.io/TizenFX/latest/api/Tizen.NUI.Components.CheckBox.html,
the CheckBox inherited https://samsung.github.io/TizenFX/latest/api/Tizen.NUI.Components.Button.html, Class Button.
So the CheckBox inherited properties of the Button also, which means CheckBox can _use_ the the properties such as 'IsEnabled', or 'Clicked' event from Button.
(If you see https://samsung.github.io/TizenFX/latest/api/Tizen.NUI.Components.CheckBox.html, you'd not see `IsEnabled` or `Clicked` at there, but they are used in the following code. )

@mgrudzinska If I misunderstood something wrong, please fix it.

Reply:
@mijin85cho yes, you are right.
The properties of the checkbox class are inherited from the button class. 

@annie-samsung, I've changed the sentence as follows:
The base class for a `CheckBox` class is a [Tizen.NUI.Components.Button](https://samsung.github.io/TizenFX/latest/api/Tizen.NUI.Components.Button.html). The `Button` properties can be used to specify the checkbox.

The button properties should be specified in the button section, so I don't want to repeat this here. 
The URL seems to be consistent with other pages of NUI guide.